tests/unit/test_stages.py

```diff
@@ -44,6 +48,79 @@ def setUp(self):
             self.block_to_test, 'user_id', mock.PropertyMock(return_value=self.user_id)
         )
 
+        self.runtime_mock.anonymous_student_id = self.user_id
+
+
+@ddt.ddt
+class TestSubmissionStage(BaseStageTest):
+    block_to_test = SubmissionStage
+
+    def _set_upload_ids(self, upload_ids):
+        submissions = []
+        for upload_id in upload_ids:
+            submission = mock.Mock()
+            submission.upload_id = upload_id
+            submissions.append(submission)
```

Comment:
I think you can replace these five lines with `submissions = [mock.Mock(upload_id=upload_id) for upload_id in upload_ids]`